### ISSUE TITLE
Feature request: multiple recognised APP_URLs

### ISSUE THREAD

**Issue by Azphreal** [BookStackApp/BookStack#268]
Very impressed with the app so far, though it took me a while to get running on a subdirectory. Now that it's up and running, I've seen that it only works on the one domain I've registered it on, which is linked to my server's internal IP. Trying to access it from the external IP doesn't render the page properly, including login, so it's effectively unusable. I am obviously able to change it, but moving between work, home, and uni and manually changing and restarting the server wears out quickly.

Is it possible to be able to list multiple sites that BookStack will load?

**Comment by ssddanbrown** [comment]
Hi @Azphreal, Supporting mulitple APP_URLS will take a fair bit of work and consideration on future features. There will always be complications trying to use mulitple base urls for a single instance of BookStack. Can you not always just use the external IP?

**Comment by Azphreal** [comment]
I figured it'd take a bit of work. Not a big deal any more, figured out a solution.

From what I remember, accessing the app on a URL it doesn't recognise doesn't properly load the CSS. The way I got around it was running two instances of the app (one using the formatted `APP_URL` for the external, one for the internal) using the same database.

In the end, for some reason my DNS is working now, so I don't even use the internal IP any more.
